hw/ip/clkmgr/doc/_index.md

```diff
@@ -48,7 +48,11 @@ The group refers to modules responsible for power up, such as power, reset and c
 Large portions of these modules operate to release clocks and resets for the rest of the design, thus cannot operate on gated versions of the clocks themselves.
 They are the only group running clocks directly from the source.
 All follow groups are derived after root clock gating.
+<<<<<<< HEAD
 See [block diagram](#block-diagram) for more details
+=======
+See [block diagram]() for more details
+>>>>>>> f8b9c9f7 ([clkmgr] clkmgr markdown creation)
```

Comment:
looks like you did not fully resolve merge conflicts here...

Reply:
fixed.

Reply:
looks like this fix made it into the subsequent commit. squashing the two commits might help to get rid of this diff.